src/main/java/frc5124/robot2020/commands/auto/runpos/DriveForTime.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc5124.robot2020.commands.auto.runpos;
+
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc5124.robot2020.subsystems.DriveTrain;
+
+public class DriveForTime extends WaitCommand {
+  DriveTrain driveTrain;
+  /**
+   * Creates a new DriveForTime.
+   */
+  public DriveForTime(DriveTrain subsystem, double time) {
+    super(time);
+    driveTrain = subsystem;
+    addRequirements(subsystem);
+  }
+
+  @Override
+  public void execute() {
+    super.execute();
+    driveTrain.tankDrive(.4, .4);
```

Comment:
could we make the drive speed parametized instead of plugging it in inside the command? My assumption when I read "Drive For Time" is that it will be full power (which is agreeably ridiculous). I think that the power choice should be made within the auto group itself.

Reply:
agreed